persist-cli/src/main/java/io/ballerina/persist/cmd/Push.java

```diff
@@ -92,8 +95,10 @@ public class Push implements BLauncherCmd {
     Project balProject;
     public String sourcePath = "";
     public String configPath = CONFIG_SCRIPT_FILE;
+    public String persistConfigPath = PERSIST_TOML_FILE;
```

Comment:
Is there a need to declare a class variable? Cant we use this straightaway in the functions?

Reply:
updated in [0b4967f](https://github.com/ballerina-platform/persist-tools/pull/54/commits/0b4967f1e318692ce31209fe0643a8f6416b6b63)